src/cascadia/inc/cppwinrt_utils.h

```diff
@@ -107,6 +107,21 @@ public:                                               \
 private:                                              \
     type _##name{ __VA_ARGS__ };
 
+// Use this macro to quickly implement both the getter and setter for an observable property.
+// This is similar to the GETSET_PROPERTY macro above, except this will also raise a
+// PropertyChanged event with the name of the property that has changed inside of the settter.
+#define DEFINE_OBSERVABLE_GETSET_PROPERTY(type, name, event)                       \

```

Comment:
This could probably just be `OBSERVABLE_GETSET_PROPERTY` - I was using `DECLARE_` and `DEFINE_` before to separate the declarations of get/set methods from their definitions, before I realized I could do it all in one macro like this

Reply:
Makes sense, took out the `DEFINE_`.